lib/owdit.js

```diff
@@ -96,7 +96,7 @@ function check(cwd, callback) {
   });
 
   if (result.status !== 0) {  //eslint-disable-line no-magic-numbers
-    return callback(new Error(`Error running npm ls: ${result.stderr}`));
+	console.error(`Error running npm ls: ${result.stderr}`);
```

Comment:
If you run the callback here an incorrect peer dependency will cause the whole process to bomb out